sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOTest.java

```diff
@@ -89,6 +90,7 @@
 
 /** Tests of {@link CassandraIO}. */
 @RunWith(JUnit4.class)
+@Ignore("Ignore until BEAM-8025 is resolved")
```

Comment:
```suggestion
@Ignore("Ignore until https://issues.apache.org/jira/browse/BEAM-8025 is resolved")
```
Just to make it clickable.

Reply:
done 👍